test/parallel/test-abort-uncaught-exception.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const spawn = require('child_process').spawn;
+const node = process.execPath;
+
+if (process.argv[2] === 'child') {
+  setImmediate(function() {
+    throw new Error('child error');
+  });
+} else {
+  run('', null);
+  run('--abort-on-uncaught-exception', 'SIGABRT');
+}
+
+function run(flags, signal) {
+  let args = [__filename, 'child'];
+  if (flags)
+    args.unshift(flags);
+
+  var child = spawn(node, args);
+  child.on('exit', function(code, sig) {
+    if (!common.isWindows) {
+      assert.strictEqual(sig, signal);
+    }
```

Comment:
If we are ignoring this assertion for windows, why don't we skip the test for windows?